src/v0/destinations/twitter_ads/transform.js

```diff
@@ -131,6 +131,20 @@ function processTrack(message, metadata, destination) {
     identifiers.push({ twclid: message.properties.twclid });
   }
 
+  if (message.properties.ip_address) {
+    const ipAddress = message.properties.ip_address.trim();
+    if (ipAddress) {
+      identifiers.push({ ip_address: ipAddress });
+    }
+  }
+
+  if (message.properties.user_agent) {
+    const userAgent = message.properties.user_agent.trim();
+    if (userAgent) {
+      identifiers.push({ user_agent: userAgent });
```

Comment:
why can't we use https://www.rudderstack.com/docs/event-spec/standard-events/common-fields/#contextual-fields:~:text=from%20the%20SDK.-,userAgent,-String